### ISSUE TITLE
marketplace.fedramp.gov nav bar update

### ISSUE THREAD

**Issue by rhoesing** [GSA/fedramp-gov#81]
In our efforts to expedite the launch of fedramp.gov we forget that the subdomain's nav bar would lose all links and not reflect the the new look of the nav bar on fedramp.gov.  The repo for the marketplace is: https://github.com/18f/fedramp-dashboard

@wslack Are we able to update mimicking the new nav on fedramp.gov?  If the timing on this will be a while I am also happy to jump into the fedramp-dashboard repo and atleast update links for now.

**Comment by rhoesing** [comment]
@cbukovac I went in to the marketplace and did a quick update to the Navbar to atleast make all the the links work and have the same nav as fedramp.gov.  I did not take time to mess with the CSS to make it look more like fedramp.gov.  But I think I can sit down and mess with it later this week.  I don't think we need to have the federalist folks mess around with this quite yet.  I'll take a shot at updating first.
